tests/system/test/MEZOTransfers.test.ts

```diff
@@ -94,6 +94,68 @@ describe("MEZOTransfers", function () {
     });
   });
 
+  describe("multipleApproveAreHandledProperlyByTheCore", function () {
```

Comment:
Maybe:
```suggestion
  describe("approveMEZOAndBTCSimultaneously", function () {
```
I think this better fits in the existing nomenclature of this test. Also the "AreHandledProperlyByTheCore" is actually the expectation and should be handled in the `it`s.